pkg/operator/internal/controllers/kubecarrier.go

```diff
@@ -73,6 +73,7 @@ func (c *KubeCarrierStrategy) GetManifests(ctx context.Context, component Compon
 	}
 	return manager.Manifests(
 		manager.Config{
+			Name:      kubeCarrier.Name,
```

Comment:
is this some bugfix? If so why wasn't it caught in the e2e tests?

Reply:
No this is not, the new `Name` field is added to the `Config` of `KubeCarrier` since the `"app.kubernetes.io/instance"` needs the name